src/tools/dashboard/components/Dashboard.tsx

```diff
@@ -39,7 +39,10 @@ const Dashboard = () => {
       <div>
         Try using <Link to={LocationToPath[Location.Blast]}>BLAST</Link>,{' '}
         <Link to={LocationToPath[Location.Align]}>Align</Link>,{' '}
-        <Link to={LocationToPath[Location.IDMap]}>ID Mapping/Retrieve</Link> or{' '}
+        <Link to={LocationToPath[Location.UploadList]}>
+          ID Mapping/Retrieve
```

Comment:
Is that kept on purpose?

Reply:
Yes - need to talk with UX about it as we need to call this functionality in a consistent way. Testing showed that mentioning "list of ids" worked well on the home page but existing users were keener on having "ID mapping" as a header. So it needs more research.